source/content.css

```diff
@@ -24,20 +24,8 @@
 }
 
 /* Remove tooltips where unnecessary */
-[aria-label^='View the whole file']::before,
-[aria-label^='View the whole file']::after,
-[aria-label^='Change this file']::before,
-[aria-label^='Change this file']::after,
-[aria-label='Delete this file']::before,
-[aria-label='Delete this file']::after,
-[aria-label='Edit this file']::before,
-[aria-label='Edit this file']::after,
 [aria-label='Edit comment']::before,
 [aria-label='Edit comment']::after,
-[aria-label='Open this file in GitHub Desktop']::before,
-[aria-label='Open this file in GitHub Desktop']::after,
-[aria-label$='branch to make changes.']::before,
-[aria-label$='branch to make changes.']::after,
```

Comment:
Since I got a thumbs up from @bfred-it on https://github.com/sindresorhus/refined-github/pull/1581#issuecomment-429579742, I went ahead and reverted all existing overrides that were hiding the tooltips.

Tooltips should look like this now 🎉 

![screen shot 2018-10-13 at 11 48 55 pm](https://user-images.githubusercontent.com/2100222/46912426-1c56a900-cf43-11e8-9669-4428aace5959.png)
![screen shot 2018-10-13 at 11 51 43 pm](https://user-images.githubusercontent.com/2100222/46912427-1cef3f80-cf43-11e8-8092-a037afbfe1fe.png)
![screen shot 2018-10-13 at 11 51 50 pm](https://user-images.githubusercontent.com/2100222/46912428-1cef3f80-cf43-11e8-85a1-620c88961d75.png)
![screen shot 2018-10-13 at 11 52 08 pm](https://user-images.githubusercontent.com/2100222/46912429-1cef3f80-cf43-11e8-84cf-ff7d971c0d39.png)
![screen shot 2018-10-13 at 11 52 25 pm](https://user-images.githubusercontent.com/2100222/46912430-1cef3f80-cf43-11e8-9e0b-549f4cc2d4d2.png)

Reply:
The last two can still be hidden. The tool tips don’t add any more information and the icons are obvious.

Reply:
> The last two can still be hidden. 

I agree 👌. While _most_ of the tooltips listed there (now removed) were added because of this specific issue within `file-header`, some other tooltips, like the notification indicator for example, can be redundant.

As for `[aria-label='Edit comment']` tooltip, it looks like there's currently a "bug" in Github (it could be intentional 🤷‍♂️) that's causing tooltips within dropdown items not to show up due to `overflow: hidden` applied on `dropdown-item`.

So I'm not sure if we should remote or keep it. It's hidden by default; with or without the extension. In case this was resolved by Github at some point, we would probably want to continue to hide it, since it's not adding any additional information.

![screen shot 2018-10-14 at 3 49 42 pm](https://user-images.githubusercontent.com/2100222/46921405-c7f60c80-cfc8-11e8-9edd-fa209e74fe28.png)

![screen_shot_2018-10-14_at_3_30_35_pm](https://user-images.githubusercontent.com/2100222/46921406-cd535700-cfc8-11e8-8b6c-a0c582a933d6.png)

![screen shot 2018-10-14 at 3 30 30 pm](https://user-images.githubusercontent.com/2100222/46921411-d6442880-cfc8-11e8-849b-c2d54a6cb723.png)

Reply:
All of these selectors should be restored, the icons are self-explanatory:

```css
[aria-label^='View the whole file']::before,
[aria-label^='View the whole file']::after,
[aria-label^='Change this file']::before,
[aria-label^='Change this file']::after,
[aria-label='Delete this file']::before,
[aria-label='Delete this file']::after,
[aria-label='Edit this file']::before,
[aria-label='Edit this file']::after,
```